performance-metrics/src/performance_tests.rs

```diff
@@ -76,6 +76,24 @@ fn direct_kernel_boot_path() -> PathBuf {
     kernel_path
 }
 
+fn remote_command(api_socket: &str, command: &str, arg: Option<&str>) -> bool {
+    let mut cmd = std::process::Command::new(clh_command("ch-remote"));
+    cmd.args([&format!("--api-socket={}", api_socket), command]);
+
+    if let Some(arg) = arg {
+        cmd.arg(arg);
+    }
+    let output = cmd.output().unwrap();
+    if output.status.success() {
+        true
+    } else {
+        eprintln!("Error running ch-remote command: {:?}", &cmd);
+        let stderr = String::from_utf8_lossy(&output.stderr);
+        eprintln!("stderr: {}", stderr);
```

Comment:
Maybe you can use one line to implement this, there are some examples in integration.rs.

Reply:
Changed.